modules/nodes-sigstore-using-validation.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-sigstore-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nodes-sigstore-using-validation_{context}"]
+= Validate cluster-wide policies using sigstore
+
+The following procedure demonstrates how to validate cluster-wide policies using Sigstore in {product-title}.
+
+.Prerequisites
+
+* Ensure that the policy on OpenShift image repositories (quay.io/openshift-release-dev/ocp-release, quay.io/openshift-release-dev/ocp-v4.0-art-dev) allows deployment of OpenShift images required for cluster operation.
+
+.Procedure
+
+. Build and Push an Image for Signing:
++
+[source,sh]
+----
+podman login quay.io
+Enter the username and password as prompted:
+Username: qiwanredhat
+Password: <your-password>
+Login Succeeded!
+----
+
+. Build and push the image:
++
+[source,sh]
+----
+echo "FROM scratch" > Dockerfile
+podman build -t quay.io/qiwanredhat/test-sigstoresigned .
+podman push quay.io/qiwanredhat/test-sigstoresigned:latest
+----
+
+. Generate a key pair for cosign:
++
+[source,sh]
+----
+cosign generate-key-pair
+Enter a password for the private key when prompted. The keys will be saved as cosign.key and cosign.pub.
+Sign the Image Using Cosign:
+Sign the image using your private key and push the signature to the registry:
+sh
+cosign sign --key ./cosign.key --registry-username=<your-username> --registry-password=<your-password> quay.io/qiwanredhat/test-sigstoresigned:latest
+----
+
+. Launch OpenShift Cluster ClusterBot to enable the feature gate:
++
+[source,sh]
+----
+launch quay.io/openshift-release-dev/ocp-release:4.16.0-rc.3-x86_64 techpreview
+----
+
+. Enable CRI-O debug log level:
+.. Create a ContainerRuntimeConfig to set the log level to debug:
```

Comment:
This is not really a part of the procedure.  It is useful to test that the thing works, sure, but users should not leave debug level output enabled forever. And, anyway, verifying that pulling an unsigned image fails is a better test.